pom.xml

```diff
@@ -167,6 +167,11 @@
         <artifactId>feign-jackson</artifactId>
         <version>${feign.version}</version>
       </dependency>
+      <dependency>
```

Comment:
Maybe we should comment here why we added the dependency, and that it can be removed once spring itself updates their dependency? Otherwise the pom just gets bigger and bigger...

Reply:
Thanks for mentioning. I agree !

Reply:
Done